diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -22,6 +22,7 @@ enum class Warnings(
     val ruleId: String,
     private val warn: String) : Rule {
     // ======== chapter 1 ========
+    MISSING_DOMAIN_NAME(false, "1.2.1", "missing domain name in config file"),
```

Comment:
Introducing a warning to check config mixes responsibilities. Let's just log a warning message like we do when config file can't be found.